### PR TITLE
Expose functionality to lookup the uv_loop_t struct size.

### PR DESCRIPTION
**Pull request by txdv**
We add a function called uv_loop_size which returns the size of the loop.
It is an additional function which complements uv_handle_size and uv_req_size
in order to provide full FFI support.

### CODE REVIEW

**Comment by txdv** [comment]
Original issue: https://github.com/joyent/libuv/issues/1334

File: include/uv.h [hunk 1]
```diff
@@ -288,6 +288,11 @@ UV_EXTERN uv_loop_t* uv_loop_new(void);
 UV_EXTERN void uv_loop_delete(uv_loop_t*);
 
 /*
+ * Returns size of the loop struct, useful for dynamic lookup with FFI
+ */
+UV_EXTERN size_t uv_loop_size();
```
Comment (saghul) [inline include/uv.h]: (void)

File: src/uv-common.c [hunk 2]
```diff
@@ -67,6 +67,11 @@ size_t uv_req_size(uv_req_type type) {
 #undef XX
 
 
+size_t uv_loop_size() {
```
Comment (saghul) [inline src/uv-common.c]: ditto.

**Comment by txdv** [comment]
Should have checked the surrounding code... But I still don't understand why we need to annotate the stuff that doesn't exist. Whatever, I will just role with the old C style.

**Comment by saghul** [comment]
IIRC it's something about C89, not sure though.

**Comment by indutny** [comment]
Hm... how do you allocate `uv_handle_t` in your FFI bindings?

**Comment by indutny** [comment]
Oops, nvm

**Comment by indutny** [comment]
Thank you, landed in e392a1a

**Comment by indutny** [comment]
And LGTM :)

**Pull request closed without merging**